### ISSUE TITLE
Could not send ICMP to service xxxxx, exit status 1

### ISSUE THREAD

**Issue by maurettoXD** [statping-ng/statping-ng#57]
### Describe the bug
Some time we receive this failure alert but from the dashboard it is not reported as a failure

### To reproduce

1. I'm using version: 0.90.78 (90b5233)
2. Tested using Webhook and Slack

![statping-pingissue-1](https://user-images.githubusercontent.com/32866094/136200184-5c7f286c-48fe-4a02-91a9-570bb315e6c9.png)

![statping-pingissue2](https://user-images.githubusercontent.com/32866094/136200212-93585892-0add-4c16-9bed-3178f2f207f6.png)

**Comment by adamboutcher** [comment]
Thanks, might be a while until this is fixed as it looks quite a weird issue.

**Comment by maurettoXD** [comment]
I just see in the source that the ping check send just 1 ping (ping -c 1) so as a workaround i make a shell script that call the original ping binary but make "N" ping (i personally set 5) and the issue is now gone... Maybe can be useful that the number of ping to send can be chosen from service settings :)

**Comment by adamboutcher** [comment]
OK, that could be a good config option, I'm guessing in your setup the first ping fails/times out due to network infrastructure.

**Comment by maurettoXD** [comment]
Yes maybe is due to network infrastructure, we are still investigating on that
